onnxruntime/test/qnn_ctx_gen/command_args_parser.cc

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) 2023 NVIDIA Corporation.
+// Licensed under the MIT License.
+
+#include "command_args_parser.h"
+
+#include <string.h>
+#include <iostream>
+#include <sstream>
+#include <string_view>
+#include <unordered_map>
+
+// Windows Specific
+#ifdef _WIN32
+#include "getopt.h"
+#include "windows.h"
+#else
+#include <unistd.h>
+#endif
+
+#include <core/graph/constants.h>
+#include <core/platform/path_lib.h>
+#include <core/optimizer/graph_transformer_level.h>
+
+#include "test_configuration.h"
+
+namespace onnxruntime {
+namespace qnnctxgen {
+
+/*static*/ void CommandLineParser::ShowUsage() {
+  printf(
```

Comment:
can you show an example commandline string which has the typical options it would have? 
e.g. would we really need to set ep.context_enable|1  ? it would be nice to set certain options by default to make it easier to use because this tool is explicitly for context generation?

Reply:
done